modules/ivc_champva/app/controllers/ivc_champva/v1/uploads_controller.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+require 'ddtrace'
+
+module IvcChampva
+  module V1
+    class UploadsController < ApplicationController
+      skip_before_action :authenticate
+      before_action :authenticate, if: :should_authenticate
```

Comment:
If you intend `should_authenticate` to always be true, then you don't need to have `if: :should_authenticate`

Also it's not necessary to have `skip_before_action :authenticate`